tests/Unit/DependencyInjection/Compiler/HashGeneratorPassTest.php

```diff
@@ -49,11 +49,20 @@ public function testConfigNoContext()
         $config = $this->getBaseConfig();
         $this->extension->load([$config], $container);
         $this->userContextListenerPass->process($container);
-        if ($container->hasDefinition('service_container')) {
-            // symfony 3.3+
-            $this->assertCount(23, $container->getDefinitions());
+        if (method_exists(\ReflectionMethod::class, 'getAttributes')) {
+            if ($container->hasDefinition('service_container')) {
+                // symfony 3.3+
```

Comment:
the minimum supported symfony version is 3.4 - could you remove the if statement and whole else block, while you are touching this?